recipes/r-lda/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = '1.4.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lda
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: lda_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/lda_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lda/lda_{{ version }}.tar.gz
+  sha256: 5606a1e1bc24706988853528023f7a004c725791ae1a7309f1aea2fc6681240f
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{native}}toolchain        # [win]
```

Comment:
You don't need tohave toolchain if you have compiler('c')

Reply:
@scopatz are you sure about this? On Windows we need to treat this differently as we need, sed and coreutils and so on.

Reply:
Currently compiler('c') is just a wrapper around toolchain. I think you can install coreutils separately.

Reply:
This expands to `m2w64-toolchain` which is separate from `toolchain`.

Reply:
Ahh true, but don't they still need compiler('c') on win

Reply:
No. `compiler('c')` is for msvc, but r packages use m2w64-gcc

Reply:
Ok, got it.